instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -53,4 +54,12 @@ class CommentSection extends React.Component {
     }
 }
 
+CommentSection.propTypes = {
+    likes: PropTypes.number,
+    comments: PropTypes.array,
+    username: PropTypes.string,
+    text: PropTypes.string,
+    timestamp: PropTypes.string
+}
```

Comment:
CommentSection is only receiving comments, likes, and timestamp in the props object passed down from PostContainer.